components/ChallengeMaterial.tsx

```diff
@@ -17,10 +18,29 @@ type ChallengeMaterialProps = {
 }
 
 export const ChallengeTitleCard: React.FC<ChallengeTitleCardProps> = props => {
+  const state = props.currentState
+  let cardStyles = 'shadow-sm border-0'
+  if (state && state === 'complete') cardStyles = 'challenge-title-card--done'
```

Comment:
`state && ` is unnecessary because you literally set it in line 21.

Reply:
Fixed :)